### PR TITLE
Installer: prompt to migrate to the wrapped CLI

### PR DESCRIPTION
**Pull request by pjcdawkins**

### CODE REVIEW

File: dist/installer.php [hunk 1]
```diff
@@ -108,6 +118,21 @@ public function run() {
 
         $this->output($this->cliName . " installer", 'heading');
 
+        if ($this->migratePrompt && !getenv($this->envPrefix . 'WRAPPED') && !$this->isCI()) {
```
Comment (akalipetis) [inline dist/installer.php:121]: Is there any chance this runs with `PLATFORM_WRAPPED=1`?

Comment (pjcdawkins) [reply dist/installer.php:121]: Good point, no

**Review by akalipetis — APPROVED** [review]
LGTM

**Review by akalipetis — APPROVED** [review]
Even better with the option to cancel the wait by pressing enter. I think we're good!

**Pull request merged**